.pre-commit-config.yaml

```diff
@@ -10,9 +10,11 @@ repos:
         # exclude helm chart templates
         exclude: ^projects/control-service/projects/helm_charts
       - id: check-json
+        exclude: (^projects/frontend/data-pipelines/gui/tsconfig.json|^projects/frontend/data-pipelines/gui/package-lock.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/.eslintrc.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.prod.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.spec.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.json|^projects/frontend/data-pipelines/gui/projects/ui/.eslintrc.json|^projects/frontend/data-pipelines/gui/projects/ui/tsconfig.app.json|^projects/frontend/data-pipelines/gui/projects/ui/tsconfig.spec.json)
```

Comment:
What is the problem? I assume it's the comments . They seem to be illegal in standard josn. 


SHoudl we execlude the whole frontend project. This cherry-picking seems like it would be hard to maitanin.

Reply:
from a prev stacked PR, rebased in post-merge